executor/executor_test.go

```diff
@@ -1872,6 +1875,34 @@ func (s *testSuite) TestIsPointGet(c *C) {
 	}
 }
 
+func (s *testSuite) TestPointGetRepeatableRead(c *C) {
+	c.Assert(failpoint.Enable("github.com/pingcap/tidb/executor/pointGetRepeatableReadTest", `return(true)`), IsNil)
+	tk1 := testkit.NewTestKit(c, s.store)
+	tk1.MustExec("use test")
+	tk1.MustExec(`create table point_get (a int, b int, c int,
+			primary key k_a(a),
+			unique key k_b(b))`)
+	tk1.MustExec("insert into point_get values (1, 1, 1)")
+	tk2 := testkit.NewTestKit(c, s.store)
+	tk2.MustExec("use test")
+
+	ctx := context.WithValue(context.Background(), "pointGetRepeatableReadTest", true)
+	go func() {
+		rs, err := tk1.Se.Execute(ctx, "select c from point_get where b = 1")
+		c.Assert(err, IsNil)
+		result := tk1.ResultSetToResultWithCtx(ctx, rs[0], Commentf("execute sql fail"))
+		result.Check(testkit.Rows("1"))
+	}()
+
+	label := debugger.Bind("point-get-g2")
+	debugger.Continue("point-get-g1")
+	debugger.Break(label)
+	tk2.MustExec("update point_get set b = 2, c = 2 where a = 1")
+	debugger.Continue("point-get-g1")
```

Comment:
continue "point-get-g2"?

Reply:
continue "point-get-g1" is right here...  it means notify the other goroutine to continiue, not this one...
Notify the `"select c from point_get where b = 1"` transaction, to continue the second get(key)